eureka-protempa-etl/src/main/java/edu/emory/cci/aiw/cvrg/eureka/etl/job/Task.java

```diff
@@ -164,8 +164,10 @@ public void run() {
 			this.etl.run(myJob, propDefArray, propIdsToShowArray, this.filter, this.updateData, this.prompts);
 			this.etl.close();
 			JobEvent completedJobEvent = new JobEvent();
+			Date jobFinishedDate = new Date();
+			myJob.setFinished(jobFinishedDate);
```

Comment:
The varaible jobFinishedDate holds the current timestamp and it is assigned to the Job's end date and job events timestamp to have them both consistent.